9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
The idea behind this empty file was that people will have explicit example where the file should be and how it should look like. Ideally there should be a comment or even some commented example of some simple use case.